css/style.css

```
@@ -15,5 +26,240 @@ body {
  * 
  * - When using Flexbox, remember the items you want to move around need to be inside a parent container set to 'display: flex'
  */
+body {
+    width:100%;
+    display:flex;
+    flex-direction: column;
+    padding: 0 auto;
+}
+header {
+    padding: 0 7%;
+    padding-top:10px;
+    display:flex;
+    justify-content: space-between;
+    align-items: center;
+}
+
+.logo {
+    width: 20px;
+}
+
+nav {
+    min-width: 50%;
+}
+
+ .navigation {
+     width:100%;
+     padding: 0 15px;
+     list-style: none;
+     display:flex;
+     justify-content: space-between;
+ }
+
+ .nav-mobile {
+    width:100%;
+    padding: 0;
+    list-style: none;
+ }
+
+ .nav-mobile li {
+    padding: 5px 0;
+ }
+
+ .navigation a, .nav-mobile a {
+    text-decoration: none;
+    color:var(--grey-dark);
+}
 
+.strong {
+    color:var(--grey-darker);
+    font-weight: 500;
+}
+
+.navigation a:hover, .nav-mobile a:hover{
+    color: var(--orange-dark);
+} 
+
+.hamburger{
+    display:none;
+}
+
+.hero {
+    background-image: url("../img/first-background.jpg");
+    background-size: cover;
+    height: 60vh;
+    display:flex;
+    align-items: center;
+    justify-content: center;
+    flex-direction: column;
+    color:var(--white);
+    text-align: center;
+    padding: 0 10%;
+}
+
+button {
+    background-color:var(--orange-dark);
+    padding: 15px 35px;
+    border-radius: 15px;
+    margin-top:30px;
+    border:0;
+    color:var(--white)
+}
+
+button:hover{
+    background-color:var(--grey-darker);
+    color:var(--orange-light);
+}
+
+.hero h1 {
+    font-weight: 400;
+    font-size:56px;
+    margin-bottom:0px;
+}
+
+.hero h2 {
+    font-weight: 400;
+    font-size:31px;
+    margin-bottom:10px;
+}
+
+.main {
+    padding:30px 7%;
+}
+
+.main h3 {
+    text-align:center;
+    font-size: 3rem;
+    font-weight: 400;
+}
+
+.feature-container {
+    display:flex;
+    justify-content: space-around;
+    padding-bottom:2rem;
+}
+
+.feature {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    font-size: 1.5rem;
+    text-align:center;
+    margin:0.5rem;
+}
```

At the moment these images are not centered on the page (i.e. the coffee mug is not sitting in the middle of the page as in the reference image). You can set a max-width for the container, and then set a specific width on each feature image to make them have the same width and be centered. Just a possible suggestion here:

```suggestion
.feature-container {
    display:flex;
    justify-content: space-around;
    padding-bottom:2rem;
    max-width: 50rem;
    margin: 0 auto;
}

.feature {
    display: flex;
    flex-direction: column;
    align-items: center;
    font-size: 1.5rem;
    text-align:center;
    margin: 0.5rem auto;
    width: 18rem;
}
```